pkg/kubelet/kubelet.go

```diff
@@ -1666,8 +1667,14 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 		metrics.PodStartLatency.Observe(metrics.SinceInMicroseconds(firstSeenTime))
 	}
 
+	var statusAnnotations map[string]string
+	if apparmor.PodUsesAppArmor(pod) && apiPodStatus.Phase != api.PodFailed {
```

Comment:
If the pod is rejected by different reason, not AppArmor related, shouldn't we mark status annotation wrongly here? I mean without such annotation.